app/views/perilune/home/index.html.haml

```diff
@@ -1,6 +1,7 @@
+%script{:src => "https://code.highcharts.com/highcharts.js"}
+%script{:src => "https://unpkg.com/alpinejs"}
```

Comment:
Blocking point. Can't figure how to inherit them from main app since this current app in a mounted engine on main app.

Reply:
Still searching.. This is the last point.